### PR TITLE
completed challenge problem

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

File: section1/set8/nestedLoops/challenge.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package main
```
Comment (goel4607) [inline section1/set8/nestedLoops/challenge.go:1]: I would appreciate if you include the goal of your problem; like what you are trying to achieve; also please what kind of i/p and o/p is expected from the program; just copy the problem statement here

File: section1/set8/nestedLoops/challenge.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+func main() {
+	n, err := strconv.Atoi(os.Args[1])
+
+	if err != nil {
+		fmt.Println("Provide an integer.")
+		return
+	}
+
+	fmt.Printf("%5s", "X")
```
Comment (goel4607) [inline section1/set8/nestedLoops/challenge.go:26]: include comments like what is the significance of these prints, for e.g. say something like printing n nums for debugging etc

File: section1/set8/nestedLoops/challenge.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	for i := 0; i <= n; i++ {
+		fmt.Printf("%5d", i)
+	}
+	fmt.Println()
+
+	for i := 0; i <= n; i++ {
+		fmt.Printf("%5d", i)
+
+		for j := 0; j <= n; j++ {
```
Comment (goel4607) [inline section1/set8/nestedLoops/challenge.go:37]: good to see that you are practicing nested loops

File: section1/set8/nestedLoops/main.go [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+func main() {
+	const max = 5
+
+	fmt.Printf("%5s", "X")
+	for i := 0; i <= max; i++ {
+		fmt.Printf("%5d", i)
+	}
+	fmt.Println()
+
+	for i := 0; i <= max; i++ {
```
Comment (goel4607) [inline section1/set8/nestedLoops/main.go:16]: please tell what is being achieved in this loop like printing the numbers to practice nested for loop

**Review by goel4607 — CHANGES_REQUESTED** [review]

File: section1/set8/nestedLoops/challenge.go (hunk 3 above)
Comment (goelvatsal) [reply section1/set8/nestedLoops/challenge.go:37]: this is the example program

Comment (goelvatsal) [reply section1/set8/nestedLoops/challenge.go:37]: i am copying down what the instructor wrote

File: section1/set8/nestedLoops/main.go (hunk 4 above)
Comment (goelvatsal) [reply section1/set8/nestedLoops/main.go:16]: Fixed and added for other for loops

File: section1/set8/nestedLoops/challenge.go (hunk 1 above)
Comment (goelvatsal) [reply section1/set8/nestedLoops/challenge.go:1]: fixed

File: section1/set8/nestedLoops/challenge.go (hunk 2 above)
Comment (goelvatsal) [reply section1/set8/nestedLoops/challenge.go:26]: fixed

Comment (goel4607) [reply section1/set8/nestedLoops/challenge.go:26]: It looks good now. Thanks

File: section1/set8/nestedLoops/challenge.go (hunk 3 above)
Comment (goel4607) [reply section1/set8/nestedLoops/challenge.go:37]: ok

**Review by goel4607 — APPROVED** [review]

**Pull request merged**